docs/kb/advanced-certificates.md

```diff
@@ -74,14 +74,15 @@ If you need to replace the default self-signed certificate, or replace an expire
 
 ### Steps
 
-Step 1 - Transfer or copy the contents of the root certificate to the VMware Event Broker Appliance
+Step 1 - Transfer or copy the contents of the root certificate to the VMware Event Broker Appliance. If you need a free, publicly signed certificate, see the [Let's Encrypt](#letsencrypt) section below.
 
+<a id="replacestep2"></a>
 Step 2 - Delete the existing Event Router TLS secret
 ```console
 kubectl -n vmware-system delete secret eventrouter-tls
 ```
 
-Step 3 - Create a new Event Router TLS secret with the new certificate keypair
+Step 3 - Create a new Event Router TLS secret with the new certificate keypair. Note that the public key filename for Let's Encrypt is `fullchain.pem`
```

Comment:
Personally I find it confusing to mention `fullchain.pem` here since is not related to the following CLI command and  confused me. (same with next section)

Reply:
As a solution, you could show  two separate `console` sections, one for custom crt, one for Let's Encrypt to make the distinction clear.

Reply:
I moved the warning down to the Let's Encrypt section, where it probably belongs.